doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc

```diff
@@ -29,3 +29,5 @@ endif::[]
 * *Mortgages*: (Decision management) Example loan approval process using decision assets. Determines loan eligibility based on applicant data and qualifications.
 * *Employee_Rostering*: (Business optimization) Example employee rostering optimization using decision and solver assets. Assigns employees to shifts based on skills.
 * *OptaCloud*: (Business optimization) Example resource allocation optimization using decision and solver assets. Assigns processes to computers with limited resources.
+* *Course_Scheduling*: (Business optimization) Example course scheduling and curriculum decision process. Assign lectures to rooms and determines a student's curriculum based on factors, such as course conflicts and class room capacity.
+* *Dinner_Party*: (Business optimization) Guest seating optimization using guided decision tables. Dinner party seating assignments based on each guest's job type, political beliefs, and known relationships.
```

Comment:
Minor suggestion:
s/Dinner party seating assignments based on each guest's job type, political beliefs, and known relationships./Assigns guest seating based on each guest's job type, political beliefs, and known relationships./

Reply:
Changed as suggested.